share/module_check_a_mundo.F

```diff
@@ -1903,6 +1903,15 @@ END FUNCTION bep_bem_nbui_max
         ENDIF
       ENDDO
 
+!-----------------------------------------------------------------------
+!   Check that max_ts_level should be smaller than the number of half levels
+!-----------------------------------------------------------------------
+   IF ( model_config_rec % max_ts_locs .gt. model_config_rec %e_vert(1)-1 )  then
```

Comment:
@smileMchen 
This should be max_ts_level, right?

Reply:
@davegill 
Yes it should be max_ts_level. I will recommit the code.